.gitmodules

```diff
@@ -9,4 +9,4 @@
 	url = https://github.com/spesmilo/electrum
 [submodule "py-ipv8"]
 	path = Tribler/pyipv8
-	url = https://github.com/Tribler/py-ipv8.git
+	url = https://github.com/devos50/py-ipv8.git
```

Comment:
❗️

Reply:
I know and this will be removed after the review and when the IPv8 PR is merged.